src/handlers/baseFieldsHandlers.ts

```diff
@@ -28,10 +33,10 @@ const getBaseFields = async (req: Request, res: Response) => {
 };
 
 const postBaseField = async (req: Request, res: Response) => {
```

Comment:
There's no longer such a thing as posting a base field -- only put (the route and openAPI spec should be removed too)